src/docs/product/performance/retention-priorities/index.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Retention Priorities
+sidebar_order: 100
+redirect_from:
+  - /product/data-management-settings/server-side-sampling/
+  - /product/data-management-settings/server-side-sampling/current-limitations/
+  - /product/data-management-settings/server-side-sampling/sampling-configurations/
+  - /product/data-management-settings/dynamic-sampling/current-limitations/
+  - /product/data-management-settings/dynamic-sampling/sampling-configurations/
+  - /product/data-management-settings/dynamic-sampling/
+  - /product/performance/performance-at-scale/
+  - /product/performance/performance-at-scale/getting-started/
+  - /product/performance-at-scale/benefits-performance-at-scale/
+description: "Learn how Sentry determines retention priorities and how to update them."
+---
+
+<Note>
+
+Retention priorities are only available on the latest version of our plans, starting from 1M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.
+
+</Note>
+
+While storing all your data makes sense at relatively low volumes, as your application scales, storing a raw copy of all your data has diminishing returns. When data reaches high volumes, Sentry begins to automatically prioritize retaining certain transactions over others with server-side data retention strategies called retention priorities.
+
+This doesn't affect your [performance metrics](https://github.com/product/performance/metrics/), which are still based on all the events you send to Sentry, ensuring that you get a full view of your application's health.
+
+## Prerequisites
+
+- Latest version of [Sentry Business plan](https://sentry.io/pricing/).
+- Admin-level permissions.
+- Latest version of one of the below SDKs:
+  - Python: 1.7.2 or later
+  - JavaScript: 7.6.0 or later
+  - Apple: 7.23.0 or later
+  - Android: 6.5.0 or later
+  - React Native: 4.3.0 or later
+  - Dart and Flutter: 6.11.0 or later
+  - PHP: 3.9.0 or later
+  - Laravel: 3.0.0 or later
+  - Symfony: 4.4.0 or later
+  - Ruby: 5.5.0 or later
+  - Java: 6.5.0 or later
+  - .NET: 3.22.0 or later
+  - Go: 0.16.0 or later
+
+## Retention Priorities
+
+Below is a list of the transactions we prioritize and deprioritize. They are enabled by default, but can be updated on a per-project basis to better fit your organization's needs.
+
+### Prioritized Transactions
+
+Sentry automatically prioritizes and retains the following types of transactions:
+
+#### Latest Release
+
+Latest releases are a retention priority because when you create a new release, we assume you'll want to have more visibility during the early adoption phase which improves your ability to catch new issues as your release is gaining traction.
+Because you may want to deploy the same release in different environments, we'll also take your environment into account and prioritize those transactions.
```

Comment:
```suggestion
Latest releases are a retention priority because when you create a new release, we assume you'll want to have more visibility during the early adoption phase, which improves your ability to catch new issues as your release is gaining traction.
Because you may want to deploy the same release in different environments, we'll also take your environment into account and prioritize those transactions.
```